Assets/Plugins/Source/EOSManager.cs

```diff
@@ -1051,7 +1051,40 @@ private void CallOnConnectLogin(Epic.OnlineServices.Connect.LoginCallbackInfo co
                     callback.Invoke(connectLoginData);
                 }
             }
+            //-------------------------------------------------------------------------
+#if UNITY_IOS
+            [DllImport("__Internal")]
+            static private extern IntPtr LoginUtility_get_app_controller();
+
+            IOSLoginOptions MakeIOSLoginOptionsFromDefualt(Epic.OnlineServices.Auth.LoginOptions loginOptions)
+            {
+                IOSLoginOptions modifiedLoginOptions = new IOSLoginOptions();
+                modifiedLoginOptions.ScopeFlags = loginOptions.ScopeFlags;
+                
+                var credentials = new IOSCredentials();
+
+                credentials.Token = loginOptions.Credentials.Value.Token;
+                credentials.Id = loginOptions.Credentials.Value.Id;
+                credentials.Type = loginOptions.Credentials.Value.Type;
+                credentials.ExternalType = loginOptions.Credentials.Value.ExternalType;
+
+                var systemAuthCredentialsOptions = new IOSCredentialsSystemAuthCredentialsOptions();
+
+                systemAuthCredentialsOptions.PresentationContextProviding = LoginUtility_get_app_controller();
+                credentials.SystemAuthCredentialsOptions = systemAuthCredentialsOptions;
 
+                modifiedLoginOptions.Credentials = credentials;
+
+                //print(loginOptions.ScopeFlags.ToString());
```

Comment:
These commented out debug lines can be deleted.

Reply:
Absolutely,  lost track of them.